backend/cmd/multiplexer.go

```diff
@@ -339,6 +350,7 @@ func (m *Multiplexer) dialWebSocket(wsURL string, tlsConfig *tls.Config, host st
 	)
 	if err != nil {
 		logger.Log(logger.LevelError, nil, err, "dialing WebSocket")
+		logger.Log(logger.LevelError, nil, resp, "WebSocker response")
```

Comment:
When I was a kid, I always wanted this: https://en.wikipedia.org/wiki/Super_Soaker

Reply:
🕸️ ⚽